docs/source/user-guide/tabular/tutorials/plot_quick_train_test_validation.py

```diff
@@ -0,0 +1,199 @@
+# -*- coding: utf-8 -*-
+"""
+Train Test Validation Scenario on Lending Club Data - 5 min Quickstart
+***********************************************************************
+
+The deepchecks train test validation suite is relevant any time you have data splits
+that you wish to validate:
+whether it's for comparing distributions across different data splits
+(e.g. before training a model or when splitting data for cross-validation),
+or for comparing new data batch to previous data batches.
+Here we'll use data from the lending club dataset, to demonstrate how you can run
```

Comment:
link to the dataset?

Reply:
added, also to dataset integrity quickstart (do you think it looks ok even though it's to API reference?)